### PR TITLE
Premium Redis Cache with Diagnostics and Alerts

### PR DESCRIPTION
**Pull request by rayapa**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@rayapa__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

**Comment by singhkay** [comment]
@rayapa There are already 4 redis premium tempaltes in a current open PR. Let's change the focus of this template to teach how to add alerts to redis cache. What do you think?

**Comment by rayapa** [comment]
Hi,
Thanks for your input. Ill change the template to focus more on adding diagnostics and alerts. Because I spent quite a bit time trying to enable the Diagnostics and alerts that should help others...
Thanks,Ray DatlaDate: Mon, 12 Oct 2015 14:42:02 -0700

**Comment by rayapa** [comment]
Hi Kay,
BTW, I have not found any templates in repository that would provision Azure Premium Redis Cache. The only templates I found related to Azure Redis Cache offering are following two.1. 101-redis-cache2.201-web-app-with-redis-cache
Thanks,Ray DatlaFrom: rayapa@hotmail.com
To: reply@reply.github.com
Subject: RE: [azure-quickstart-templates] Premium Redis Cache with Diagnostics and Alerts (#706)
Date: Mon, 12 Oct 2015 20:15:54 -0400




Hi,
Thanks for your input. Ill change the template to focus more on adding diagnostics and alerts. Because I spent quite a bit time trying to enable the Diagnostics and alerts that should help others...
Thanks,Ray Datla

Date: Mon, 12 Oct 2015 14:42:02 -0700

**Comment by singhkay** [comment]
@rayapa They're still in the PR I linked https://github.com/Azure/azure-quickstart-templates/pull/686/files

File: 201-diagnostics-alerts-on-premium-redis/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+        "description": "The name of the Azure Redis Cache to create."
+      }
+    },
+    "redisCacheLocation": {
+      "type": "string",
+      "metadata": {
+        "description": "The location of the Redics Cache. For best perfomance, use the same location as the app to be used with the cache."
+      }
+    },
+    "diagnosticsStorageAccountName": {
```
Comment (singhkay) [inline 201-diagnostics-alerts-on-premium-redis/azuredeploy.json]: Suggestion: For any parameters that refer to existing resources, please prefix "existing" to parameter names

**Comment by singhkay** [comment]
@rayapa Any update on this?

**Comment by singhkay** [comment]
@rayapa closing due to lack of activity. Feel free to reopen with an update

**Pull request closed without merging**